hudi-common/src/main/java/org/apache/hudi/common/util/CleanerUtils.java

```diff
@@ -28,19 +28,19 @@
 import org.apache.hudi.common.versioning.clean.CleanV1MigrationHandler;
 import org.apache.hudi.common.versioning.clean.CleanV2MigrationHandler;
 
-import com.google.common.collect.ImmutableMap;
-
 import java.io.IOException;
 import java.util.List;
 
+import static org.apache.hudi.common.util.CollectionUtils.Maps;
+
 public class CleanerUtils {
   public static final Integer CLEAN_METADATA_VERSION_1 = CleanV1MigrationHandler.VERSION;
   public static final Integer CLEAN_METADATA_VERSION_2 = CleanV2MigrationHandler.VERSION;
   public static final Integer LATEST_CLEAN_METADATA_VERSION = CLEAN_METADATA_VERSION_2;
 
   public static HoodieCleanMetadata convertCleanMetadata(HoodieTableMetaClient metaClient,
       String startCleanTime, Option<Long> durationInMs, List<HoodieCleanStat> cleanStats) {
-    ImmutableMap.Builder<String, HoodieCleanPartitionMetadata> partitionMetadataBuilder = ImmutableMap.builder();
+    Maps.MapBuilder<String, HoodieCleanPartitionMetadata> partitionMetadataBuilder = new Maps.MapBuilder<>();
```

Comment:
this probably can be just replaced by a concrete map impl in place right? this sort of usage does not really warrant the builder pattern IMO.